timetracker/entries/templates/base.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>{% block title %}{% endblock %}</title>
+
+    <style media="screen">
+      .container {
+        width: 40em;
+        margin: 0 auto;
+        font-family: 'Helvetica', 'Arial', sans-serif;
+      }
+
+      .nav {
+        background-color: #121212;
+        color: #eee;
+      }
+
+      ul.nav {
+        list-style: none;
+        background-color: #444;
+        text-align: center;
+        padding: 0;
+        margin: 0;
+      }
+      .nav li {
+        font-family: 'Oswald', sans-serif;
+        font-size: 1.2em;
+        line-height: 40px;
+        height: 40px;
+        display: inline-block;
+        width: 120px;
+      }
+
+      .nav a {
+        text-decoration: none;
+        color: #fff;
+        display: block;
+        transition: .3s background-color;
+      }
+
+      .nav a:hover {
+        background-color: #0C4B33;
+        
+      }
+
+      .nav a.active {
+        background-color: #fff;
+        color: #444;
+        cursor: default;
+      }
+      .wrapper {
+        margin: 0 auto;
+        padding: 40px;
+        max-width: 800px;
+      }
+
+      .table {
+        margin: 0 0 40px 0;
+        width: 100%;
+        box-shadow: 0 1px 3px rgba(0, 0, 0, 0.2);
+        display: table;
+      }
+      @media screen and (max-width: 580px) {
+        .table {
+          display: block;
+        }
+      }
+
+      .row {
+        display: table-row;
+        background: #f6f6f6;
+      }
+      .row:nth-of-type(odd) {
+        background: #e9e9e9;
+      }
+      .row.header {
+        font-weight: 900;
+        color: #ffffff;
+        background: #ea6153;
+      }
+      .row.green {
+        background: #27ae60;
+      }
+      .row.blue {
+        background: #2980b9;
+      }
+      @media screen and (max-width: 580px) {
+        .row {
+          padding: 8px 0;
+          display: block;
+        }
+      }
+
+      .cell {
+        padding: 6px 12px;
+        display: table-cell;
+      }
+      @media screen and (max-width: 580px) {
+        .cell {
+          padding: 2px 12px;
+          display: block;
+        }
+      }
+
+    </style>
+  </head>
+  <body>
+    <div class="container">
+      <ul class="nav">
+        <li><a href="{% url 'entries' %}" class="
+          {% if entry_list %}active{% endif %}">Entries</a></li>
+        
+        <li><a href="{% url 'clients' %}"class="{% if client_list %}active{% endif %}">Clients</a></li>
+        
+        <li><a href="{% url 'projects' %}" class="{% if project_list %}active{% endif %}">Projects</a></li>
+      </ul>
+      <!-- these blocks will only render when their respective context  is provided -->
+      {% block entries %}{% endblock %}
```

Comment:
There's no need to have a separate block for each child template.

Since only one child can be using this at a time, they can all override the same block name.